packages/app/src/app/store/modules/workspace/actions.js

```diff
@@ -47,7 +82,12 @@ export function updateSandbox({ api, state }) {
     .put(`/sandboxes/${sandboxId}`, body)
     .then(data => {
       window.history.pushState({}, null, sandboxUrl(data));
-      return { data };
+      return {
```

Comment:
The data returned here now is different, I'm not sure if that's supposed to happen. We also automatically camelize the keys of the response.

Reply:
fixed